src/rust/node-identifier/src/dynamic_sessiondb.rs

```diff
@@ -79,7 +79,7 @@ where
 
         let put_req = PutItemInput {
             item: serde_dynamodb::to_hashmap(&mapping).unwrap(),
-            table_name: "static_mapping_table".to_owned(),
+            table_name: std::env::var("STATIC_MAPPING_TABLE").expect("STATIC_MAPPING_TABLE"),
```

Comment:
this should probably go in grapl-config, also we need to make sure this is being passed into the service as an env var in `docker-compose.yml`

Reply:
Yeah, good catch, have a docker-compose.yml locally.

I'll look into grapl-config.

Reply:
@inickles-grapl yes that sounds right